kubernetes/charts/edge-kubernetes/templates/iotedged-deployment.yaml

```diff
@@ -32,7 +32,11 @@ spec:
           {{- if .Values.iotedged.data.httpsProxy }}
           - name: "https_proxy"
             value: {{ .Values.iotedged.data.httpsProxy | quote}}
-          {{end}}
+          {{- end}}
+          {{- if .Values.iotedged.data.noProxy }}
+          - name: "no_proxy"
+            value: {{ .Values.iotedged.data.noProxy | quote}}
+          {{- end}}
```

Comment:
Do we need to add HTTP_PROXY as well so we cover all proxy env variables?

Reply:
Not for edgeAgent or iotedged - they only make https requests.  

Reference: https://docs.microsoft.com/en-us/azure/iot-edge/how-to-configure-proxy-support#configure-the-iot-edge-agent